### DISCUSSION TITLE
How do I remap arrow keys on specific websites?

### DISCUSSION THREAD

**Discussion by luckycold** [gdh1995/vimium-c#1156] [Q&A]
I'm trying to remap h and l for left and right arrow for youtube but I get an error when I attempt to map the left and right key.

Here's the config:
```
env youtube host="https://www.youtube.com/"

map h runKey expect={"youtube":"<v-hyoutube>"}
mapKey <v-hyoutube> <left>
```
with an error saying `Error: mapKey: a source key should be a single key with an optional mode id: mapKey <v-hyoutube> left.`

I'm probably doing this wrong. But I thought it might also be possible that it's a limitation of trying to use the arrow keys.

**Comment by gdh1995** [comment]
There are indeed some issues in your example:
1. "mapKey" is used before "map", which means "runKey"'s expected value should only be the "key sequence" part of a "map" rule.
2. "mapKey" can not change how webpages treat pressed keys - it only affects Vimium C itself.


If you want to change a current time of a video, then you may use javascript code to do so, and the 2nd line will look like:
map <v-hyoutube> openUrl url="javascript:<your-code-here>"

As for detailed code, there have been a few issues related, you may search in this repo.
